.npmrc

```diff
@@ -1,2 +1,5 @@
-legacy-peer-deps=true
-engine-strict=true
+save-exact = true
```

Comment:
While working on various dependencies, I noticed that this results in the default `^` prefix for compatible versions being removed. I am not sure how I feel about that, but I wanted to flag it as a key side-effect.